README.md

```diff
@@ -34,11 +34,21 @@ Feel free to try out and let us know if you cannot wait.
 
 ## Install dockert3kit into your distribution
 
-Add `lauri/dockert3kit` as dev dependency in your composer, using the latest stable release is highly recommended.
+Add `lauri/dockert3kit` as dev dependency in your composer.
+
+***Note***
+
+**If you are using EXT:solr = 6.0.0 please use the 6.0.0 release**
+
+*Example*:
+
+    composer require --dev lauri/dockert3kit '~6.0.0'
+
+**If you are using EXT:solr = 6.1.0 please use the 6.1.0 release**
 
 *Example*:
 
-    composer require --dev lauri/dockert3kit '~3.0.0'
+    composer require --dev lauri/dockert3kit '~6.1.0'
```

Comment:
@laurisaarni  ok, I change release 6.0.0 to 4.0.0 and 6.1.0 to 5.0.0
What about the  docker image tags do I change that as well, right now it is using
saarni/solr-for-typo3:6.0.0 for typo3 solr 6.0.0
and
saarni/solr-for-typo3:6.1.0 for typo3 solr 6.1.0